azubi-timesheet.py

```diff
@@ -135,13 +162,35 @@ def parse_cli(args=None):
     :rtype: :class:`argparse.Namespace`
     """
     parser=argparse.ArgumentParser(description=__doc__,
-                                     prog="azubi-timesheet",
+                                     prog=__prog__,
                                      add_help=False)
     parser.add_argument("-v", "--version",
                         action="version",
-                        version="%(prog)s v0.9",
+                        version="%(prog)s {}".format(__version__),
                         help="Show program's version number and exit."
                         )
+    # toms: Maybe better create subparsers for this.
+    # For an example, look at:
+    # https://gist.github.com/tomschr/8bfb228b68c52e8b6686213d89c842d5
+    # Python doc:
+    # https://docs.python.org/3/library/argparse.html#argparse.ArgumentParser.add_subparsers
```

Comment:
Subparsers are cool. This way, there will be a help message for every subcommand :)